app/models/bus_stop.rb

```diff
@@ -245,6 +245,10 @@ def decide_if_completed_by(user)
     end
   end
 
+  def add_hastus_id_to_name
+    self.name = name + " (#{hastus_id})"
```

Comment:
I don't agree that we should be editing the objects. Again - this is a *presentational* concern, not a *data* concern. Let's have this be a presentational method only, not one that edits the objects.